src/test/test-mount-util.c

```diff
@@ -69,6 +69,12 @@ static void test_mount_option_mangle(void) {
         opts = mfree(opts);
 
         assert_se(mount_option_mangle("rw,relatime,fmask=0022,dmask=0022,\"hogehoge", MS_RDONLY, &f, &opts) < 0);
+
+        assert_se(mount_option_mangle("mode=1777,size=10%,nr_inodes=400k,uid=496107520,gid=496107520,context=\"sys.id:sys.role:systemd.nspawn.container.fs:s0,s1\"", 0, &f, &opts) == 0);
+        assert_se(f == 0);
+        log_info("%s", opts);
+        assert_se(streq(opts, "mode=1777,size=10%,nr_inodes=400k,uid=496107520,gid=496107520,context=\"sys.id:sys.role:systemd.nspawn.container.fs:s0,s1\""));
+        opts = mfree(opts);
 }
```

Comment:
Probably best to use '\"system_u:object_r:svirt_sandbox_file_t:s0:c0,c1\"' here if this test runs on Fedora?

Reply:
haha, this is just a test of option string handling, so the actual content is not important. But yeah, will fix. Thanks.

Reply:
Updated.